com.github.sdbg.debug.ui/plugin.xml

```diff
@@ -1077,6 +1079,15 @@
             <bundle name="com.github.sdbg.debug.core"/>
       </component>
    </extension>
+   
+   <!-- Used to launch chrome from another plugin like GWT -->
+   <extension
+         point="com.github.sdbg.debug.ui.launchChrome">
+      <extensions
+            class="com.github.sdbg.ui.launchers.ChromeLaunchShortcutExt">
+      </extensions>
+   </extension>
+
```

Comment:
Hm.  How isolated is chrome launcher?  It makes sense to me to have SDBG depend on Gwt instead of the other way around.  I think what Ivan says about not needing the extension point is correct.  If the launcher is in an exported package, we should be able to create it directly, should we not?

If you can access the interface, you should be able to new the launcher...

And, if Gwt defined the extension point, then here we could get the array of IConfigurationElement of all supplied launchers, and SDBG could simply provide all that it cared too.  Given the faster release cycle of SDBG, it makes sense; Gwt plugin gets updated once, and SDBG can add launchers whenever one if ready.

Even better, this would allow other plugins to add a gwt-related launcher that can, for example, run a pre-processor before launching Gwt runtime.